app/Module/LanguageFrench.php

```diff
@@ -183,60 +207,71 @@ static function (int $up, int $down, string $sex) use ($symmetricCousin, $compou
             Relationship::fixed('beau-père', '%s du beau-père')->parent()->husband(),
             Relationship::fixed('beau-parent', '%s du beau-parent')->parent()->married()->spouse(),
             Relationship::fixed('belle-fille', '%s de la belle-fille')->married()->spouse()->daughter(),
-            Relationship::fixed('beau-fils', '%s du beau-fils')->married()->spouse()->son(),
+            Relationship::fixed('gendre', '%s du beau-fils')->married()->spouse()->son(),
```

Comment:
Is this change right?  It is inconsistent with the lines around it?

Reply:
Thanks for spotting that, this is a mistake indeed. The line should not have been changed at all (there is a change further down from beau-fils to gendre, but that applies only to the in-laws context).
I have resubmitted a corrected file.